ngraph/python/tests/__init__.py

```diff
@@ -173,3 +171,6 @@ def xfail_test(reason="Mark the test as expected to fail", strict=True):
 xfail_issue_49752 = xfail_test(reason="RuntimeError: Unsupported dynamic ops: v1::Pad")
 xfail_issue_49753 = xfail_test(reason="RuntimeError: Unsupported dynamic ops: v1::StridedSlice")
 xfail_issue_49754 = xfail_test(reason="RuntimeError: Unsupported dynamic ops: v1::TopKIE")
+xfail_issue_49913 = xfail_test(reason="CPU supports Gelu with tanh mode only")
```

Comment:
CPU plugin already supports Gelu with both modes. @a-sidorova, why we have an xfail here? May be some nGraph transformation from Gelu7 to Gelu1 was not disabled for the CPU plugin?
Or this is a merge issue?

Reply:
right, that was a merge issue